src/templates/layers-list.html

```diff
@@ -36,7 +36,7 @@ <h3 class="toolbox-title layers-title">Layers
   <script type="text/template" id="layer-item-template">
     <li class="layer-item {{isselected:current-layer-item}}"
         data-layer-index="{{layerindex}}">
-        {{layername}}
+        <span class="layer-name">{{layername}}</span>
```

Comment:
Everything looks good! let's just add title="{{layername}}" here and it should be good to go :)

Thanks!